### DISCUSSION TITLE
第5节课笔记(1班)

### DISCUSSION THREAD

**Discussion by AXYZdong** [InternLM/Tutorial#346] [笔记]
请在下方提交第 5 节课笔记链接，知乎/CSDN/飞书 等等都可。
笔记让知识记得更牢固💪

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/13/ai-internlm-lmdeploy/

**Comment by gongxiucheng** [comment]
https://ckfajwu6zv.feishu.cn/docx/GuhFdjcnCoMJaZxCePbcA3zdnLg?from=from_copylink

**Comment by Gu-Xiao** [comment]
https://zhuanlan.zhihu.com/p/677456098

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135596348

**Comment by woxinyonghen** [comment]
https://blog.csdn.net/xingchengyonghen/article/details/135570022

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135611214

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/B4rFdjWMuoiQXwxqqVuc5waynlb#KHyYdDplDoJeDVxz3F5c41iPnlc

**Comment by strong-leaf** [comment]
https://zhuanlan.zhihu.com/p/677990123

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135637903

**Comment by 10086zgyd** [comment]
https://blog.csdn.net/weixin_55546799/article/details/135671688

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/fwttpmeg4niheegy?singleDoc# 《lesson5 LMDeploy 的量化和部署 笔记》

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135682163

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/chap4.md

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/678053451

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/679062776

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/1601447672b34e6baafec6dd38dbc8d8
